src/StripeResource.ts

```diff
@@ -1,10 +1,14 @@
-import * as utils from './utils';
+import {
+  getDataFromArgs,
+  getOptionsFromArgs,
+  makeURLInterpolator,
+  protoExtend,
+  stringifyRequestData,
+} from './utils.js';
+import {stripeMethod} from './StripeMethod.js';
 
 // Provide extension mechanism for Stripe Resource Sub-Classes
-StripeResource.extend = utils.protoExtend;
-
-// Expose method-creator
-StripeResource.method = require('./StripeMethod');
```

Comment:
Is this method visible through `stripe` instance? Like `Stripe.StripeResource.method`?

Reply:
Something doesn't have to be exported directly to be visible to customers and we should make sure things are not broken.

Reply:
Yes that was visible on `StripeResource`, adding it back. Thanks for catching that!